src/main/java/org/ays/common/model/response/AysResponse.java

```diff
@@ -36,8 +43,11 @@ public class AysResponse<T> {
     /**
      * A pre-defined success response with no content.
      */
```

Comment:
Javadoc değişmemiş gibi görünüyor

Reply:
```suggestion
 /**
     * A success response with no content.
     */
```

Uygun mudur bu şekilde ?

Reply:
Uygundur eline sağlık ✅